tensorflow/core/api_def/base_api/api_def_UniqueV2.pbtxt

```diff
@@ -41,19 +41,19 @@ In other words, for an `1-D` tensor `x` with `axis = None:
 For example:
 
 ```
-# tensor 'x' is [1, 1, 2, 4, 4, 4, 7, 8, 8]
-y, idx = unique(x)
+x = tf.constant([1, 1, 2, 4, 4, 4, 7, 8, 8])
+y, idx = tf.raw_ops.UniqueV2(x=x, axis=[0])
 y ==> [1, 2, 4, 7, 8]
 idx ==> [0, 0, 1, 2, 2, 2, 3, 4, 4]
 ```
 
 For an `2-D` tensor `x` with `axis = 0`:
 
 ```
-# tensor 'x' is [[1, 0, 0],
-#                [1, 0, 0],
-#                [2, 0, 0]]
-y, idx = unique(x, axis=0)
+x = tf.constant([[1, 0, 0],
```

Comment:
While at it, could you please add the example as a testable docstring?

E.g.
>>>  x = tf.constant[1, 0, 0],
...                               [1, 0, 0],
...                               [2, 0, 0]]
>>> y, idx = tf.raw_ops.UniqueV2(x=x, axis=[0])
>>> print(y)
tf.Tensor(
[[1 0 0]
 [2 0 0]], shape=(2, 3), dtype=int32)
>>> print(idx)
tf.Tensor([0 0 1], shape=(3,), dtype=int32)



Here are a couple of examples: 
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/api_def/base_api/api_def_Tile.pbtxt
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/api_def/base_api/api_def_Sign.pbtxt

Reply:
Please don't use docstrings in base_api/pbtxt. To make them testable docstrings you have to extract them to Python files first.

Reply:
I added an example at https://github.com/tensorflow/tensorflow/pull/57873#issuecomment-1331485417

Reply:
Ah, please ignore my comment about testable doc string.
Thanks for pointing it out Mihai.